packages/indy-vdr/src/dids/IndyVdrSovDidResolver.ts

```diff
@@ -69,7 +73,9 @@ export class IndyVdrSovDidResolver implements DidResolver {
       )
       const response = await pool.submitReadRequest(request)
 
-      if (!response.result.data) return {}
+      if (!response.result.data) {
+        return
```

Comment:
Maybe we can return `null` here to indicate there's not value (return type of `{} | void` feels a bit weird)